lib/core/base/context.js

```diff
@@ -91,9 +92,9 @@ function normalizeContext(context) {
       return {
         include:
           context.include && +context.include.length
-            ? context.include
+            ? Array.from(context.include)
             : [document],
-        exclude: context.exclude || []
+        exclude: context.exclude ? clone(context.exclude) : []
```

Comment:
This isn't going to work. Exclude can contain DOM nodes. We can't clone DOM elements.

Reply:
Looks like DOM nodes aren't being cloned.